x11-misc/xkbd/xkbd-0.8.16-r1.ebuild

```diff
@@ -42,13 +38,11 @@ src_configure() {
 
 src_compile() {
 	default
-
 	use doc && docbook2html README
```

Comment:
docbook2html command seems to fail with a lot of error messages, but it produces some working html files. What should be done about it. Remove die or nuke whole doc useflag?